src/Tools/dotnet-monitor/CollectionRules/Options/Triggers/AspNetResponseStatusOptions.cs

```diff
@@ -31,7 +31,7 @@ internal sealed class AspNetResponseStatusOptions :
             Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_AspNetResponseStatusOptions_ResponseCount))]
         [Required]
         [Range(1, int.MaxValue)]
-        public int ResponseCount { get; set; }
+        public int? ResponseCount { get; set; }
```

Comment:
Since there is now validation on these properties, can they stay non-nullable? Nullable and [Required] are a bit confusing.

Reply:
So, if I revert to making them non-nullable, the error message you'll get for omitting them is "The field `RequestCount` must be between 1 and 2147483647". I guess the question is, do we want an error message that says the field is required, or just that the field must be in that range? I can imagine this being a slight source of confusion if I see that error message and think that doesn't make sense since I didn't set it at all, because I assumed there was a default (as opposed to the "default" just being that of an int, which is 0). I'll defer to what you all think makes more sense though.

Reply:
I'm guessing that the schema.json (which you'll need to regenerate for this change; otherwise it will fail the json schema tests) will say that type of this property will be `[ "integer", "null" ]` but the parent object will have it be required. This means that:
```json
"Trigger": {
    "Type": "AspNetRequestCount",
    "Settings": {
        "RequestCount": null
    }
},
```
is allowed by the schema, which isn't what we want.

If the property remains non-nullable and I got a message stating "The field RequestCount must be between 1 and 2147483647", while I think it's mildly strange that I wasn't told it was required, it's easy to understand what the mitigation is.

Reply:
I would be ok with non-nullable even if the error message is a bit indirect.